docs-site/docs/user/project/projects/bot-accounts/manage-bot-accounts.md

```diff
@@ -4,10 +4,65 @@ id: manage-bot-accounts
 title: Manage Bot Accounts
 ---
 
-<!-- TODO: Add headers with description/explanations of following:
-- Add Bot Account
-- Remove Bot Account
-- Add Membership
-- Remove Membership
-Note: A group bot that is created in a group can be added as a member to other projects/groups (global). However, a project bot can only be associated to the project it was created within
--->
+A project bot can only be associated to the project it was created within.
+
+## Add Bot Account
+
+Prerequisite:
+
+- You must have at least a **Maintainer** role.
+
+To add a bot account to a project:
+
+1. From the left sidebar, select **Projects** and find your project.
+2. From the left sidebar, select **Settings > Bot Accounts**.
+3. Click the **New bot account** button.
+4. Enter the token name in the **Name** field.
+5. Select an access level.
+6. Set an optional expiry date.
+7. Select at least one scope.
+8. Click the **Submit** button.
+
+You should now see the new bot account listed within the bot accounts table. Above the table you should also see a section for the newly created personal access token. Copy the personal access token, as it cannot be accessed afterwards.
+
+## Remove Bot Account
+
+Prerequisites:
+
+- You must have at least a **Maintainer** role.
+
+To remove an existing bot account from a project:
+
+1. From the left sidebar, select **Projects** and find your project.
+2. From the left sidebar, select **Settings > Bot Accounts**.
+3. On the right hand side of the row for the bot account you want to remove, click the **Remove** link.
+4. Confirm that you would like to remove the bot account by clicking the **Confirm** button.
+
+## Add Membership
+
+Prerequisite:
+
+- You must have at least a **Maintainer** role.
+
+To add a bot account membership to a project:
+
+1. From the left sidebar, select **Projects** and find your project.
+2. From the left sidebar, select **Members**.
+3. Click the **Add Member** button.
+4. Select the bot account you want to add to the project.
+5. Select an access level (role).
+6. Select an optional **Access expiration**.
+7. Click the **Add member to project** button.
+
+## Remove Membership
+
+Prerequisites:
+
+- You must have at least a **Maintainer** role.
+
+To remove a bot account membership from a project:
+
+1. From the left sidebar, select **Projects** and find your project.
+2. From the left sidebar, select **Members**.
+3. On the right hand side of the row for the bot account membership you want to remove, click the **Remove** link.
+4. Confirm that you would like to remove the bot account membership by clicking the **Confirm** button.
```

Comment:
Also make a note here if the Bot Account was deleted from the Settings -> Bot Accounts page that the membership would automatically be removed. This will also apply to the groups -> manage bot accounts docs

Reply:
30ed197bcd870d4b31e3b30cf22454ef0bee829e